### PR TITLE
Update links and prose on Learn page

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: app/templates/learn/index.hbs [hunk 1]
```diff
@@ -79,27 +79,22 @@
...
+        <strong>Ember CLI:</strong> The official command line toolkit to develop Ember applications. Check out the <a href="https://cli.emberjs.com/release/" rel="nofollow noopener" target="_blank">Ember CLI Guides</a>, and the documentation on how to <a href="https://cli.emberjs.com/release/advanced-use/blueprints/" rel="nofollow noopener" target="_blank">extend Ember CLI</a> as an addon developer.
       </li>
-
       <li>
         <strong>Ember Inspector:</strong> A browser plugin/bookmarklet that helps you inspect and debug applications. Learn how best to use it in the <a href="https://guides.emberjs.com/release/ember-inspector/installation/" rel="nofollow noopener" target="_blank">Ember Inspector Guides</a>.
       </li>
-
-      <li>
-        <strong>Liquid Fire:</strong> A toolkit to add animations and transitions to your application. Check out their <a href="https://ember-animation.github.io/liquid-fire/" rel="nofollow noopener" target="_blank">interactive documentation</a>.
```
Comment (MelSumner) [inline app/templates/learn/index.hbs:90]: Ember Animation should be here.

**Comment by MelSumner** [comment]
@locks I'd love to move this PR forward when you get a chance.

File: app/templates/learn/index.hbs (hunk 1 above)
Comment (locks) [reply app/templates/learn/index.hbs:90]: Resolved below!

**Comment by locks** [comment]
@MelSumner resolved ember-animated and the conflict with master!

**Comment by locks** [comment]
Thank you for the reviews, merged!

**Pull request merged**